docs/engineering/release-gates/local-only-policy.md

```
@@ -0,0 +1,123 @@
+the remote:
+
+```
+ln -sf ../../docs/engineering/release-gates/pre-push-hook.sh .git/hooks/pre-push
+```
+
+(Symlink so future updates to the canonical hook flow through automatically.)
+
+To bypass the hook for a known-safe push (CI-skip, docs-only, etc.) without
+removing it:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)